src/main/java/blackjack/controller/Game.java

```diff
@@ -0,0 +1,117 @@
+package blackjack.controller;
+
+import blackjack.domain.Dealer;
+import blackjack.domain.Player;
+import blackjack.domain.Players;
+import blackjack.domain.Result;
+import blackjack.domain.ResultStatus;
+import blackjack.view.InputView;
+import blackjack.view.OutputView;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+public class Game {
+
+    private static final int BLACKJACK_SCORE = 21;
+    private final Dealer dealer;
+    private final Players players;
+
+
+    public Game(Dealer dealer, Players players) {
+        this.dealer = dealer;
+        this.players = players;
+    }
+
+    public void play() {
+        initializeHands();
+
+        List<Player> players = this.players.getPlayers();
```

Comment:
```suggestion
        List<Player> players = players.getPlayers();
```
큰 문제는 아니지만 `this` 는 여기서 없어도 될 것 같아요.
그리고 `players` 라는 이름의 인스턴스 변수가 이미 Game 에 있다보니 지역변수에 같은 이름을 지으면 실수할 가능성이 높아져요.
다른 이름으로 바꿔보는건 어떨까요?